### PR TITLE
swagger: update

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

Update swagger in accordance to all on our changes

Closes: #5011

### CODE REVIEW

File: rpc/swagger/swagger.yaml [hunk 1]
```diff
@@ -1362,7 +1362,7 @@ components:
             type: string
             example: "Listener(@)"
         n_peers:
-          type: number
+          type: integer
           example: "1"
```
Comment (cmwaters) [inline rpc/swagger/swagger.yaml:1366]: Is this right? looks like a string in the example

File: rpc/swagger/swagger.yaml [hunk 2]
```diff
@@ -1615,112 +1616,121 @@ components:
...
                           example: "Q29zbW9zaGkgTmV0b3dva28="
+                        index:
+                          type: boolean
+                          example: true
             end_block:
               type: "array"
-              x-nullable: true
+              nullable: true
               items:
                 type: "object"
```
Comment (cmwaters) [inline rpc/swagger/swagger.yaml:1711]: do we need to remove the quotation marks for these types?

File: rpc/swagger/swagger.yaml [hunk 3]
```diff
@@ -2052,21 +2066,33 @@ components:
...
+                          type: string
+                          example: "100000"
+                        max_age_duration:
+                          type: string
+                          example: "100000"
+                        max_num:
+                          type: integer
+                          example: 50
+                        proof_trial_period:
+                          type: string
```
Comment (cmwaters) [inline rpc/swagger/swagger.yaml:2095]: you could also add the defaults here if you wanted - don't know if it is necessary though

File: rpc/swagger/swagger.yaml (hunk 2 above)
Comment (marbar3778) [reply rpc/swagger/swagger.yaml:1711]: I was going to make it more consistent and refactor things in a different PR, but it works and renders correctly like this.

File: rpc/swagger/swagger.yaml (hunk 3 above)
Comment (marbar3778) [reply rpc/swagger/swagger.yaml:2095]: The example is shown an in a JSON object example so default would not make a difference here

**Pull request merged**